presto-delta/src/main/java/com/facebook/presto/delta/DeltaSplitManager.java

```diff
@@ -72,9 +73,11 @@ public ConnectorSplitSource getSplits(
         private final DeltaTable deltaTable;
         private final CloseableIterator<AddFile> fileIterator;
         private final int maxBatchSize;
+        private final ConnectorSession session;
```

Comment:
do we need this member?

Reply:
session is required for fetching out node_selection_strategy session property